momentumnet/plot_memory.py

```diff
@@ -12,7 +12,7 @@
 
```

Comment:
why the new file address?

Reply:
should not be at the root

Reply:
it takes a long time for this example to run with circle ci so i moved it for now